src/System.Net.Security/tests/FunctionalTests/NegotiateStreamTestForUnix.cs

```diff
@@ -50,18 +53,29 @@ public void Dispose()
         // on the host. Returns true available, false otherwise
         public bool CheckAndInitializeKerberos()
         {
-            if (_isKrbInstalled)
```

Comment:
i am planning to cherry pick this change in master image.. and resend the PR against master..